config/v1/types_infrastructure.go

```diff
@@ -705,7 +705,45 @@ type AlibabaCloudResourceTag struct {
 
 // NutanixPlatformSpec holds the desired state of the Nutanix infrastructure provider.
 // This only includes fields that can be modified in the cluster.
-type NutanixPlatformSpec struct{}
+type NutanixPlatformSpec struct {
+	// prismCentral holds the endpoint address and port to access the Nutanix Prism Central
+	// +kubebuilder:validation:Required
+	PrismCentral NutanixPrismEndpoint `json:"prismCentral"`
+
+	// prismElements holds one or more endpoint address and port data to access the Nutanix
+	// Prism Elements (clusters) of the Nutanix Prism Central. Currently we only support one
+	// Prism Element (cluster) for an openshift cluster, where all the Nutanix resources (VMs, subnet, etc.)
+	// used in the Openshift cluster locate. In the future, we may support the Nutanix resources (VMs, etc.)
+	// used in the Openshift cluster can come from multiple Prism Elements (clusters) of the Prism Cental.
```

Comment:
Nit: You have mixed the casing of `OpenShift` within this paragraph, you can choose either `openshift` or the preferred `OpenShift`, but you need to make it consistent, this doc string is user facing.

Also, I think this will read better, wdyt?
```suggestion
	// used in the OpenShift cluster are located. In the future, we may support Nutanix resources (VMs, etc.)
	// spread over multiple Prism Elements (clusters) of the Prism Central.
```